surfactant/infoextractors/docker_image.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC
+# See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+import gzip
+import json
+import subprocess
+import tempfile
+
+from loguru import logger
+
+import surfactant.plugin
+from surfactant.sbomtypes import SBOM, Software
+
+
+def is_docker_scout_installed():
+    # Check that Docker Scout can be run
+    result = subprocess.run(["docker", "scout", "--help"], capture_output=True, check=False)
```

Comment:
From the CI tests, it looks like `subprocess.run` can sometimes throw a `FileNotFoundException` (are there any others?) if the `docker` command isn't installed on a system.